controllers/velero.go

```diff
@@ -286,32 +147,28 @@ func (r *DPAReconciler) buildVeleroDeployment(veleroDeployment *appsv1.Deploymen
 	if veleroDeployment == nil {
 		return fmt.Errorf("velero deployment cannot be nil")
 	}
+	// Auto corrects DPA
+	dpa.AutoCorrect()
 
-	//check if CSI plugin is added in spec
-	for _, plugin := range dpa.Spec.Configuration.Velero.DefaultPlugins {
-		if plugin == oadpv1alpha1.DefaultPluginCSI {
-			// CSI plugin is added so ensure that CSI feature flags is set
-			dpa.Spec.Configuration.Velero.FeatureFlags = append(dpa.Spec.Configuration.Velero.FeatureFlags, enableCSIFeatureFlag)
-			break
-		}
-	}
 	_, err := r.ReconcileRestoreResourcesVersionPriority(dpa)
 	if err != nil {
 		return fmt.Errorf("error creating configmap for restore resource version priority:" + err.Error())
 	}
-
+	// orig := veleroDeployment.DeepCopy()
```

Comment:
lets remove this comment as we have debug mode now.

Reply:
+1

Reply:
```suggestion
```